portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,26 @@
 <html>
```

Comment:
Add the DOCTYPE here: go/htmlcssstyle#Document_Type

Reply:
I unfortunately don't have permission to view that style guide. The one they gave us is here: https://google.github.io/styleguide/... I'm not sure whether it's the same or not.
The guide I have says I just need the `<!DOCTYPE html>` tag. Is there something else I should add?

Reply:
That was all.

Reply:
Got it! It looks like it's already in there from before.

Reply:
The review page wasn't loading the first line originally. Sorry about that.